guide/popular-topics/audit-logs.md

```diff
@@ -146,7 +146,7 @@ client.login('your-token-goes-here');
 As was the case in the previous examples, we can see what happened, to whom it happened, but not who executed the action. Enter once again audit logs where we will limit ourselves to 1 entry, and looking at the `MEMBER_BAN_ADD` type. Our `guildBanAdd` listener then becomes.
 
 ```js
-client.on('guildBaneAdd', async message => {
+client.on('guildBanAdd', async message => {
```

Comment:
Hey uhm... doesn't the `guildBanAdd` event take `Guild` as the first parameter?

Reply:
Looks like similiar thing also happens on line 97.

Reply:
resolved